frontend/public/locales/en/letters.json

```
@@ -1,9 +1,9 @@
 {
   "index": {
-    "page-title": "View my letters",
+    "page-title": "Letters",
     "breadcrumbs": {
       "home": "Home",
-      "view-letter": "View letters"
+      "letters": "letters"
```

```suggestion
      "letters": "Letters"
```

(same correction needed for `/fr/letters.json`